tests/field_deconstruction/tests.py

```diff
@@ -196,53 +196,53 @@ def test_float_field(self):
     def test_foreign_key(self):
         # Test basic pointing
         from django.contrib.auth.models import Permission
-        field = models.ForeignKey("auth.Permission", models.CASCADE)
+        field = models.ForeignKey("auth.permission", models.CASCADE)
```

Comment:
Why this change?

Reply:
Unnecessary, undone, thanks for spotting.